programming_examples/basic/passthrough_kernel/run.lit

```diff
@@ -6,7 +6,8 @@
 // RUN: xchesscc_wrapper aie2 -I %aietools/include -DBIT_WIDTH=8 -c %S/../../../aie_kernels/aie_generic/passThrough.cc -o passThrough.cc.o
 // RUN: %python %S/aie2.py 4096 | aie-opt -cse -canonicalize -o ./aie.mlir
 // RUN: %python aiecc.py --xbridge --aie-generate-cdo --aie-generate-ipu --no-compile-host --xclbin-name=aie.xclbin --ipu-insts-name=insts.txt ./aie.mlir
-// RUN: clang %S/test.cpp -o test.exe -std=c++11 -Wall -DPASSTHROUGH_SIZE=4096 -I%S/../../../runtime_lib/test_lib %S/../../../runtime_lib/test_lib/test_utils.cpp %xrt_flags -lrt -lstdc++ -lboost_program_options -lboost_filesystem
-// RUN: %run_on_ipu ./test.exe -x aie.xclbin -k MLIR_AIE -i insts.txt | FileCheck %s
+// RUN: g++-13 %S/test.cpp -o test.exe -std=c++23 -Wall -DPASSTHROUGH_SIZE=4096 -I%S/../../../runtime_lib/test_lib %S/../../../runtime_lib/test_lib/test_utils.cpp %xrt_flags -lrt -lstdc++ -lboost_program_options -lboost_filesystem
+// RUN: XRT_HACK_UNSECURE_LOADING_XCLBIN=1 ./test.exe -x aie.xclbin -k MLIR_AIE -i insts.txt | FileCheck %s
```

Comment:
In [main](https://github.com/Xilinx/mlir-aie/blob/main/utils/run_on_ipu.sh) you can keep `run_on_ipu` to get the envvar + it sources the xrt setup script for you (and it no longer does signing). The alternative is to source the setup script in the workflow yaml.